doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+  We set `block=True`,
+  so that ``Experiment.start()`` waits until the last Model has finished
+  before returning: it will act like a job monitor, letting us know
+  if processes run, complete, or fail.
+
+Clobber the databases
+^^^^^^^^^^^^^^^^^^^^^
+Finally, use the ``Experiment.stop()`` function to stop the database instances. Print the
+workflow summary with ``Experiment.summary()``.
+
+.. code-block:: python
+
+  exp.stop(single_shard_db, multi_shard_db)
+  logger.info(exp.summary())
+
+When you run the experiment, the following output will appear.
+
+.. code-block:: bash
+
+  00:00:00 system.host.com SmartSim[#####]INFO
+  |    | Name                         | Entity-Type   | JobID       | RunID   | Time    | Status    | Returncode   |
+  |----|------------------------------|---------------|-------------|---------|---------|-----------|--------------|
+  | 1  | colo_model                   | Model         | 1538905.8   | 0       | 3.5465  | Completed | 0            |
+  | 2  | single_shard_db_identifier_0 | DBNode        | 1538905.5   | 0       | 73.1798 | Cancelled | 0            |
+  | 3  | multi-shard-db-identifier    | DBNode        | 1538905.6+2 | 0       | 49.8503 | Cancelled | 0            |
+
+Source Code
+-----------
+
+Experiment script
+^^^^^^^^^^^^^^^^^
+.. code-block:: python
+
+  import numpy as np
+  from smartredis import Client
+  from smartsim import Experiment
+  from smartsim.log import get_logger
+  import sys
+
+  exe_ex = sys.executable
+  logger = get_logger("Multidb Experiment Log")
+  exp = Experiment("getting-started-multidb", launcher="auto")
+
+  single_shard_db = exp.create_database(port=6379, db_nodes=1, interface="ib0", db_identifier="single_shard_db_identifier")
+  exp.generate(single_shard_db, overwrite=True)
+
+  multi_shard_db = exp.create_database(port=6380, db_nodes=3, interface="ib0", db_identifier="multi-shard-db-identifier")
+  exp.generate(multi_shard_db, overwrite=True)
+
+  exp.start(single_shard_db, multi_shard_db)
+
+  driver_client_single_shard = Client(cluster=False, address=single_shard_db.get_address()[0], logger_name="Single shard db logger")
+  driver_client_multi_shard = Client(cluster=True, address=multi_shard_db.get_address()[0], logger_name="Multi shard db logger")
+
+  array_1 = np.array([1, 2, 3, 4])
+  driver_client_single_shard.put_tensor("tensor_1", array_1)
+
+  array_2 = np.array([5, 6, 7, 8])
+  driver_client_multi_shard.put_tensor("tensor_2", array_2)
+
+  check_single_shard_db_tensor_incorrect = driver_client_single_shard.key_exists("tensor_2")
+  check_multi_shard_db_tensor_incorrect = driver_client_multi_shard.key_exists("tensor_1")
+  logger.info(f"The multi shard array key exists in the incorrect database: {check_single_shard_db_tensor_incorrect}")
+  logger.info(f"The single shard array key exists in the incorrect database: {check_multi_shard_db_tensor_incorrect}")
+
+  model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/model_ex.py")
```

Comment:
make the `exe_args` here point to a fairly generic path and make a comment that people will have to change this line for things to work